doc/specs/stdlib_experimental_stats.md

```diff
@@ -6,6 +6,54 @@ title: experimental_stats
 
 [TOC]
 
+## `corr` - Pearson correlation of array elements
+
+### Description
+
+Returns the Pearson correlation of the elements of `array` along dimension `dim` if the corresponding element in `mask` is `true`.
+
+The Pearson correlation between two rows (or columns), say `x` and `y`, of `array` is defined as:
+
+```
+ corr(x, y) = cov(x, y) / sqrt( var(x) * var(y))
+```
+
+### Syntax
+
+`result = [[stdlib_experimental_stats(module):corr(interface)]](array, dim [, mask])`
```

Comment:
What is the "syntax" of this syntax? What does `[[stdlib_experimental_stats(module):corr(interface)]]` mean? I think this subsection and snippet should show me how to call `corr()`. I don't think this is doing that. Shouldn't it be this:

```suggestion
`result = corr(array, dim[, mask])`
```

Reply:
Oh, is this some special FORD syntax for hyperlinking? If yes, please ignore this comment.

Reply:
It is for FORD. It creates a link to the module.
See an example for `cov` [here](https://stdlib.fortran-lang.org/page/specs/stdlib_experimental_stats.html#syntax).